plugin.js

```diff
@@ -12,16 +12,17 @@ function rawBody (fastify, opts, next) {
     return
   }
 
-  const { field, encoding, global, runFirst, routes } = Object.assign({
+  const { field, encoding, global, runFirst, routes, jsonContentTypes } = Object.assign({
     field: 'rawBody',
     encoding: 'utf8',
     global: true,
     runFirst: false,
-    routes: []
+    routes: [],
+    jsonContentTypes: ['application/json'],
```

Comment:
A minor linting fix before merge!

```suggestion
    jsonContentTypes: ['application/json']
```